modules/PersonalBank.py

```diff
@@ -27,35 +29,38 @@ def show_transaction_history(history_of_transactions):
     
         sorted_list = sorted(history_of_transactions, 
                 key = lambda Transaction: Transaction.date)
-        for i in sorted_list:
-            print(i)
+        return sorted_list
 
     def __repr__(self):
         
-        return "({}, {})".format(self.balance, self.history_of_transactions)
+        return "{}".format(self.balance)
         
         
         
 if __name__ == "__main__":
 
-    beer = Product("Beer", 3.5)     #Creating a transaction
+    beer = Product("Beer", 3.5)                 #Creating a transaction
     pizza = Product("Pizza", 7)
     a_date = datetime.date(2018, 10, 2)
     a_product_list = [beer, pizza]
     a_transaction = Transaction(a_product_list, "friends", "in", a_date)
 
-    coffee = Product("Coffee", 3)   #Creating a second transaction
+    coffee = Product("Coffee", 3)               #Creating a second transaction
     b_date = datetime.date(2018, 10, 1)
     b_product_list = coffee
     b_transaction = Transaction(b_product_list, "significant other", "out", b_date)
     
     a_history_of_transactions = []
```

Comment:
This variable should be an instance variable and not just manipulated with an instance.
For example,

```python
p = PersonalBank()
print(p.history_of_transactions)  # this should print []
```